### ISSUE TITLE
Cut error

### ISSUE THREAD

**Issue by Pullusb** [GDQuest/blender-power-sequencer#432]
**I'm submitting a...**

- [ X] Bug report.
- [ ] Feature request.

## Bug report ##

**What is the current behavior?**

When pressing `k` over a selected strip to cut at cursor  position I get following error:

```
Error: Traceback (most recent call last):
  File "G:\LOGICIELS\3D\blender-2.83\2.83\scripts\addons\power_sequencer\operators\cut_strips_under_cursor.py", line 52, in invoke
    return self.execute(context)
  File "G:\LOGICIELS\3D\blender-2.83\2.83\scripts\addons\power_sequencer\operators\cut_strips_under_cursor.py", line 65, in execute
    return bpy.ops.sequencer.cut(frame=context.scene.frame_current, side=self.side)
  File "G:\LOGICIELS\3D\blender-2.83\2.83\scripts\modules\bpy\ops.py", line 201, in __call__
    ret = op_call(self.idname_py(), None, kw)
AttributeError: Calling operator "bpy.ops.sequencer.cut" error, could not be found
```

I'm using **Power sequencer 1.4.0** (shipped with the version)
Last buildbot Blender version
Date : 2020-05-27
Hash : 79e529c5ec16
Branch : master
Version: 2.83 (sub 17)

Looking in the interactive console, I cant find `bpy.ops.sequencer.cut` with autocomplete.

**Comment by NathanLovato** [comment]
This is already fixed in master on this repository and in Blender-addons master. I'll see with the developers if we can include that in the Blender 2.83 branch.

**Comment by NathanLovato** [comment]
I've asked a developer for information. Seems we're in bcon 4 already so it might not be possible to have the fix in Blender 2.83.
